tk-demos/widget

```diff
@@ -1,5 +1,6 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/env perl
```

Comment:
given 
https://metacpan.org/source/SREZIC/Tk-804.034/demos/widget
I would prefer to either remove that line at all or make it similar